### DISCUSSION TITLE
Resources to Learn for good contribution ??

### DISCUSSION THREAD

**Discussion by Prathamix77** [mozilla/rhino#1415] [Q&A] [answered]
What is the stuff i need to know before i dive into the project and actually make good contribution to help up the Project RHINO :)

**Comment by p-bakker** [comment, marked as answer]
Tnx for you interest!

Starting points would be our readme, particularly the [Contributing PRs](https://github.com/mozilla/rhino#contributing-prs), [Code Formatting](https://github.com/mozilla/rhino#code-formatting) and [Building](https://github.com/mozilla/rhino#building) sections.

Rhino comes with an extensive testsuite, including [Test262](https://github.com/tc39/test262) integration. Check [Running Tests](https://github.com/mozilla/rhino/blob/master/testsrc/README.md) for more information. 

Next, I'd take a look at https://github.com/mozilla/rhino/contribute for a curated list of good first issues, if you don't already have particular fixes/improvements/features that you want to work on.

Hope this helps and looking forward to your contributions!
